Common/Brokerages/BinanceBrokerageModel.cs

```diff
@@ -88,6 +91,50 @@ public override IFeeModel GetFeeModel(Security security)
             return new BinanceFeeModel();
         }
 
+        /// <summary>
+        /// Binance does not support update of orders
+        /// </summary>
+        /// <param name="security"></param>
+        /// <param name="order"></param>
+        /// <param name="request"></param>
+        /// <param name="message"></param>
+        /// <returns></returns>
```

Comment:
Missing docs, let's review all the changes for missing docs 👍 

> using static QuantConnect.StringExtensions; & using System.Linq;

Not required, let's review all changes for none required imports